pkg/controllers/status.go

```diff
@@ -115,12 +115,24 @@ func (r *ClusterOperatorStatusClient) setStatusAvailable(ctx context.Context) er
 		return err
 	}
 
+	// we want to preserve the upgradeable flag in case the sync controller has marked us as not upgradeable
+	upgradeable := configv1.ConditionTrue
+	upgReason := ReasonAsExpected
+	upgMessage := ""
```

Comment:
nit: would be nice to follow [this pattern][1] and include some encouraging words here like `No concerns about updating.`, so users don't have to look at the machine-oriented `reason` to understand what's going on.  Although the `status` is pretty clear, so no worries if you actively prefer to leave `message` empty.

[1]: https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/188d382d413cb1bd2a37b2b7dc17d0d58430634e/pkg/controllers/cloud_config_sync_controller.go#L283-L287

Reply:
i was more just following the pattern of the previous code